Useful Calculator.py

```diff
@@ -1,80 +1,108 @@
 
 import math
 import random
 
-#define functions
-def clearline(n):       #this is the function for clearing lines
-    for x in range(n):  #for every number between zero and the integer provided when calling the function
-        print()         #print an empty line 
-
-def surarea():#this is the function for calculating surface area
-	
-	print("\n\n\nHere is a list of currently coded shapes.\n'rectangle' 'circle' 'triangle'\n\n\n") #this is a list of shapes that the program can calculate surface area for
-	shape = input("What is the shape you are trying to calculate?   ")                              #asks for a shape to calculate
-	if shape == "rectangle":                                                                        #checks if the shape is a rectangle  
-		l = int(input("What is the length?   "))                                                    #asks for the length of the rectangle
-		h = int(input("What is the height?   "))                                                    #asks for the height of the rectangle
-		sa = l*h                                                                                    #surface area equals length times height
-		print("\n\nThe surface area of the rectangle is " + str(sa) + " units squared.\n\n")        #print the answer in a formatted text
-	elif shape == "circle":                                                                         #checks if the shape is a circle
-		pi = math.pi                                                                                #this is pi 
-		r = int(input("What is the radius of the circle?   "))                                      #asks for the radius of the circle
-		sa = pi*(r**2)                                                                              #surface area equals pi x (radius squared)
-		print("\n\nThe surface area of the circle is " + str(sa) + " units squared.\n\n")           #print the answer in formatted text
-	elif shape == "triangle":                                                                       #checks if the shape is a triangle
-		b = int(input("What is the base of the triangle?   "))                                      #asks for the base of the triangle
-		h = int(input("What is the height of the triangle?   "))                                    #asks for the height of the triangle
-		a = b*h/2                                                                                   #area equals base times height divided by two
-		print("\n\nThe surface area of the triangle is "+ str(a) +" units squared.\n\n")            #print the answer in formatted text
-	else:                                                                                           #if the response didnt match any of the answers do the following
-		print("\n\nThis shape is not added yet please let the author know.\n\n")                    #Invalid text text
-		return
+
+# define functions
+def clearline(n):
+    """this is the function for clearing lines"""
```

Comment:
This is Docstring

Reply:
I know what it is. To me it makes the code have redundant lines and its messy. All code that i write is optimized for raspberry pi in the future. So i just try to keep the lines and file size down. Also as for long lines of code. I write and run in Notepad++ on my laptop and I can enable word wrap so long lines of code don't bother me